agent/runner/jobs/pbm_helpers.go

```diff
@@ -163,6 +163,21 @@ type pbmError struct {
 	Error string `json:"Error"`
 }
 
+// pbmConfigFlags groups the flags/options for configuring PBM.
+type pbmConfigFlag struct {
+	configFilePath string
+	forceResync    bool
+	dbURL          *url.URL
+}
+
+func createPBMConfigFlag(confFile string, dbURL *url.URL, forceResync bool) pbmConfigFlag {
+	return pbmConfigFlag{
+		forceResync:    forceResync,
+		configFilePath: confFile,
+		dbURL:          dbURL,
+	}
+}
```

Comment:
Not sure we need this func. We can fill structure in-line without this helper.

Reply:
I think using this function gives us guarantee that all fields are set as intended.

Reply:
Usually we're using Validator for that. I just think this func guarantees that values provided but doesn't check they're correct. So not a big difference.